tests/src/Kernel/Entity/GetBundleByBundleTest.php

```diff
@@ -0,0 +1,557 @@
+<?php
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\block_content\Entity\BlockContentType;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelper;
+
+/**
+ * Tests retrieving group (content) bundles by group (content) bundles.
+ *
+ * @group og
+ * @coversDefaultClass \Drupal\og\GroupManager
+ */
+class GetBundleByBundleTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'block_content',
```

Comment:
I started out with `entity_test` but it was not suitable - it has bundles but no bundle type entity, meaning it doesn't have any info about which bundles are created. It's important to retrieve bundle info for this PR, so I went for the next best thing instead :)